### PR TITLE
build and publish node

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -114,15 +123,64 @@
...
+        single-fast = pkgs.writeShellApplication rec {
+          name = "single-fast";
+          text = ''
+            ${pkgs.lib.meta.getExe golden-gate-node} --dev  
+          '';
+        };
+
+        # we do not use existing Dotsama tools as they target relay + parachains
+        # here we can evolve into generating arion/systemd/podman/k8s output (what ever will fit) easy 
+        multi-fast = pkgs.writeShellApplication rec {
```
Comment (akorchyn) [inline flake.nix:158]: Can you tell me how to run it? Just curious

File: flake.nix [hunk 2]
```diff
@@ -75,18 +75,27 @@
...
+          cargoExtraArgs = "--package ${pname} --target wasm32-unknown-unknown --no-default-features --features=aura,with-rocksdb-weights";
           RUSTFLAGS =
             "-Clink-arg=--export=__heap_base -Clink-arg=--import-memory";
-          pname = "golden-gate-runtime";
           version = "0.1.0";
+        };
 
+
+        common-native-release-attrs = common-attrs // rec {
+          cargoExtraArgs = "--package ${pname}";
```
[the remark this answers is not in the document]
Comment (akorchyn) [reply flake.nix:89]: nvm, i see now

File: flake.nix (hunk 1 above)
Comment (akorchyn) [reply flake.nix:158]: nix run .#multi-fast for curious ones.

Though, I think it's better to specify ports:
9001, 9002, 9003. 

cause sometimes you need to be connected to specific node

File: flake.nix [hunk 3]
```diff
@@ -114,15 +123,64 @@
...
+          name = "prune-running";
+          text = ''
+            pkill golden-gate-nod 
+          '';
+        };
       in
       rec {
         packages = flake-utils.lib.flattenTree {
-          inherit golden-gate-runtime;
+          inherit golden-gate-runtime golden-gate-node single-fast multi-fast;
```
Comment (akorchyn) [inline flake.nix:181]: you have missed prune-running

**Comment by akorchyn** [comment]
I think we can skip the job if md files are changed only.
paths-ignore:
      - '**/README.md'

**Comment by akorchyn** [comment]
I dont know. Seems to much run 6 minute job for PR containing only MD files.

**Review by akorchyn — APPROVED** [review]

**Review by xcthulhu — APPROVED** [review]

**Pull request merged**